evap/contributor/forms.py

```diff
@@ -25,7 +25,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.fields['general_questions'].queryset = Questionnaire.objects.filter(is_for_contributors=False).filter(
+        self.fields['general_questions'].queryset = Questionnaire.objects.general_questionnaires().filter(
```

Comment:
we're already calling those "course questionnaires" somewhere in the code.

...actually, maybe the general contribution should be called "course contribution"? but oh well...